### DISCUSSION TITLE
Distinction When Creating a Parent for a new Parent

### DISCUSSION THREAD

**Discussion by carlosmbe** [specify/specify7#5835] [Feature Requests]
Let's say the user is creating a new Geography through Data Entry and they're creating a new Parent and the overlay comes up.

The issue is that if the new parent needs a parent we get a new overlay directly on top of the previous one that looks exactly the same as the last one just with a different name.

At first thought, you'd assume that it was an overwritten. However, after saving(or moving the overlay). It appears that they were just stacked. 

It'd be great if we could communicate the stacking nature either through an animation or by moving the new overlay slightly to the side.

Focus on the Name section to follow the screen recording.

https://github.com/specify/specify7/assets/53784701/58d31766-ae55-458e-a42a-c996d1837e55

**Comment by CarolineDenis** [comment]
@maxpatiiuk , what is you opinion/advise for this?

**Comment by maxpatiiuk** [comment]
Moving the new window a bit to the side would be neat

this is how windows solved this problem too - https://youtu.be/5BZLz21ZS_Y?si=4CuR6GjPELsWeC0l&t=4 😊

But, I don't know how often this happens in real world scenarios - don't think people are creating tree parents in the form - if they are creating parents, that is usually done though the tree viewer (if only a few parents) or the workbench (if a lot of parents), so this is a low priority

Thus, until [Grant's issue list is empty](https://github.com/specify/specify7/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Grant%27s+issue+list%22), this should be iceboxed
